SUMMARY.md

```diff
@@ -223,6 +223,7 @@
 - [Regulations](libs/regulations/README.md)
 - [Residence History](libs/residence-history/README.md)
 - [Service Portal](libs/service-portal/README.md)
+    - [Service Portal Access Control Delegations](libs/service-portal/access-control/delegations/README.md)
```

Comment:
Should mark the library as hidden from gitbook (see README.md in other service portal modules).